pkg/cmd/controller/serviceaccount.go

```diff
@@ -1,86 +1,56 @@
 package controller
 
 import (
-	"k8s.io/klog/v2"
-
-	kapiv1 "k8s.io/api/core/v1"
-	sacontroller "k8s.io/kubernetes/pkg/controller/serviceaccount"
+	"fmt"
 
-	serviceaccountcontrollers "github.com/openshift/openshift-controller-manager/pkg/serviceaccounts/controllers"
+	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/controller/serviceaccount"
 )
 
 func RunServiceAccountController(ctx *ControllerContext) (bool, error) {
```

Comment:
> dead function now?

No & Yes.

No. I've left it in to not lose the original functionality. Someone can still add arbitrary ServiceAccount names to the config to be created on namespace creation. 
Yes. Effectively, for our purposes since the `len(Config.ServiceAccount.ManagedNames)` is now `0` by default.

Reply:
Sounds like it should be safely dropped, if we need it we can always bring it back, but until then - remove.